assets/app/scripts/services/data.js

```diff
@@ -704,7 +704,10 @@ angular.module('openshiftConsole')
   DataService.prototype._uniqueKeyForResourceContext = function(resource, context) {
     // Note: when we start handling selecting multiple projects this
     // will change to include all relevant scope
-    if (context.project && context.project.metadata) {
+    if (resource === "projects") { // when we are loading the projects list we dont need additional context
+      return resource;
+    }
+    else if (context.project && context.project.metadata) {
```

Comment:
I think `context.namespace` should take priority... can you debug the places we look up things from the shared namespace and make sure their cache key is correct?